Makefile

```diff
@@ -4,7 +4,8 @@ REGISTRY_TAG ?= devel
 CONTROLLER_IMAGE ?= ${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-controller:${REGISTRY_TAG}
 API_IMAGE ?= ${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-api:${REGISTRY_TAG}
 VALIDATION_IMAGE ?= ${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-validation:${REGISTRY_TAG}
-VIRT_V2V_IMAGE ?= ${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-virt-v2v:${REGISTRY_TAG}
+VIRT_V2V_IMAGE_COLD ?= ${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-virt-v2v:${REGISTRY_TAG}
```

Comment:
```suggestion
VIRT_V2V_IMAGE ?= ${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-virt-v2v:${REGISTRY_TAG}
```

Reply:
I would like to have a clear distinction what is the meaning of a variable so they cannot be confused. Where there is `VIRT_V2V_IMAGE_COLD` it refers to a single image that is used for cold migration. Where there is `VIRT_V2V_IMAGE` it refers to our internal variable that can have a combination of two images (cold and warm).

Reply:
I am thinking that this change should be temporary, so I was suggesting the minimal changes :).

Reply:
I find the way the "VIRT_V2V_IMAGE" variable is used already confusing (being able to put two different images separated by '|') and I think nobody will understand it without reading the documentation first. So I would add the warm target with the warm variable and leave the rest of the code untouched, this is, with `--action_env VIRT_V2V_IMAGE=${VIRT_V2V_IMAGE}` in the `build-operator-bundle-image` target.

Reply:
After discussing it offline I am ok with the changes the way they are

Reply:
`VIRT_V2V_IMAGE` is for internal use so the requirement on reading the documentation is not a bad thing. I understand it is already confusing to some extent, but we should find a good balance and try to reduce the confusion if it means just a few more (simple) changes.